src/resolvers/hosts/hosts.integration.ts

```diff
@@ -227,6 +230,29 @@ describe('hosts query', function () {
             expect(data).toMatchSnapshot();
         });
 
+        test('group_name ASC', async () => {
+            const { data, status } = await runQuery(BASIC_QUERY, {
+                order_by: 'group_name',
+                order_how: 'ASC'
+            });
+
+            expect(status).toEqual(200);
+            expect(data).toMatchSnapshot();
+            // the first host is not expected to have any groups, i.e. null
+            data.hosts.data[0].groups.length === null;
```

Comment:
In other parts of the code, it checks that `groups` is null, or that `groups.length` is 0. Is validating a null length the proper check here?

Reply:
Hmmm, I don't know how it passed tests.  It should have been `data.hosts.data[0].groups.data.length === 0`

Reply:
It didn't fail because that line isn't an assertion - I left a comment on the new version of the code, so I'll resolve this one